R/modelFitting.R

```diff
@@ -128,9 +129,18 @@ compBayes <- function(dat,
       })
     ptmTemp <- oneDat$ptmID #save this from the first model
   }
-
+  #how to normalize
+  if (normalize) {
+    normalize_plex <- lapply(1:length(dat), function(x) (all(dat[[x]][3,] == 0)))
+    }
+  if (normalize & normalize_ptm) {
+    normalize_plex <- lapply(1:length(dat), function(x) (TRUE))
+    }
```

Comment:
I think you could restructure to these conditionals to be a little bit cleaner, perhaps something like this:
```
if (normalize) {
normalize_plex <- ifelse(normalize_ptm, lapply(1:length(dat), function(x) (TRUE)), lapply(1:length(dat), function(x) (all(dat[[x]][3,] == 0))))
} else {
normalize_plex <- ifelse(normalize_ptm, lapply(1:length(dat), function(x) (FALSE)), dat)
}
```
At the very least, maybe find a way to avoid
```
if ( ! (normalize | normalize_ptm)){
```